sphinx_needs/directives/needsequence.py

```diff
@@ -211,6 +211,15 @@ def process_needsequence(app: Sphinx, doctree: nodes.document, fromdocname: str,
         if current_needsequence["show_filters"]:
             content.append(get_filter_para(current_needsequence))
 
+        if app.config.needs_plantuml_debug:
+            plantuml_debug(
+                "needflow",
+                app.config.plantuml,
+                puml_node.children[0]["uml"],
+                current_needsequence["docname"],
+                current_needsequence["lineno"],
```

Comment:
I think just parse the puml_node to the function, which already contains the source information